PowerPointLabs/Test/FunctionalTest/ShapesLabTest.cs

```diff
@@ -97,18 +85,29 @@ private void TestSaveShapesToShapesLabWithAddShapesButton(IShapesLabController s
             shapesLab.ClickAddShapeButton();
 
             Microsoft.Office.Interop.PowerPoint.Slide actualSlide = PpOperations.SelectSlide(7);
-            IShapesLabLabeledThumbnail addedThumbnail = shapesLab.GetLabeledThumbnail("selectMeNow1");
-            addedThumbnail.FinishNameEdit();
-            // add shapes back
-            DoubleClick(addedThumbnail as Control);
-            Microsoft.Office.Interop.PowerPoint.ShapeRange shapes = PpOperations.SelectShapesByPrefix("Group selectMeNow1");
-            Assert.IsTrue(shapes.Count > 0, "Failed to add shapes from Shapes Lab." +
-                                            "UI test is flaky, pls re-run.");
+            AddShapesToSlideFromShapesLab(shapesLab, "selectMeNow1", "Group selectMeNow1");
 
             Microsoft.Office.Interop.PowerPoint.Slide expSlide = PpOperations.SelectSlide(8);
 
             SlideUtil.IsSameLooking(expSlide, actualSlide);
             SlideUtil.IsSameAnimations(expSlide, actualSlide);
         }
+
+        private void AddShapesToSlideFromShapesLab(IShapesLabController shapesLab, string shapeThumbnail, string expectedShapePrefix) 
+        {
+            IShapesLabLabeledThumbnail thumbnail = shapesLab.GetLabeledThumbnail(shapeThumbnail);
+            thumbnail.FinishNameEdit();
+            // Add shapes from Shapes Lab to slide by double clicking
+            DoubleClick(thumbnail as Control);
+            Microsoft.Office.Interop.PowerPoint.ShapeRange shapes = PpOperations.SelectShapesByPrefix(expectedShapePrefix);
+            Assert.IsTrue(shapes.Count > 0, "Failed to add shapes from Shapes Lab." +
+                                            "UI test is flaky, pls re-run.");
```

Comment:
@initialshl I have refactored this test. I copied the comment from the older TestSaveShapesToShapesLab test, but I'm not sure why it is flaky. It has not failed for me before. Do you think I should leave the comment here?

Reply:
I think the comment can be removed since it doesnt fail.